### ISSUE TITLE
SampleConsumer blank, not receiving produced messages

### ISSUE THREAD

**Issue by sachabarber** [awslabs/amazon-kinesis-client-net#28]
So I am new to AWS Kinesis, and I have the sample basis producer working just fine with using my default profile which has my IAM user details in the store.

I can see that the producer send messages to the single stream shard. That's all good.

I downloaded the sample consumer from here and have done the following

- Made sure I had environment variables for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY for the SampleConsumer app
- Made sure I had Java installed
- Edited the SampleConsumer command line args to include "--properties kcl.properties --execute"
- Made sure the kcl.properties file has the correct streamName/executableName/regionName (which I changed to match my region for producer/my AWS account, so eu-west-2)
- Then when I run the SampleConsumer from above with the SampleProducer, all I see for the SampleConsumer is a blank console application.

The producer is 100% fine.

Anyone have any ideas?

**Comment by ohildur** [comment]
I'm on Mac OS and have a similar issue. It fails with:
No executable found matching command "dotnet-SampleConsumer.dll"

**Comment by sachabarber** [comment]
Wow really, I tried with more of the partial path. I will try that full path later too

**Comment by sachabarber** [comment]
Yeah putting in full path in kcl.properties did not fix it. I still get this

The application to execute does not exist: 'C:\Users\sacha\Downloads\amazon-kinesis-client-net-master\amazon-kinesis-client-net-master\SampleConsumer\UserssachaDownloadsamazon-kinesis-client-net-masteramazon-kinesis-client-net-masterSampleConsumerbinDebug

Anyone got any ideas of how to fix this, this is driving me crazy

**Comment by sachabarber** [comment]
Ok finally got this to work, fiddled around a bit more, turns out the IAM user I was using also needed dynamoDB permission. Though path thing above was main issue

**Comment by ericbrumfield** [comment]
For what it's worth, I think the issues in this thread are related to path problems for the bootstrap executableName.  I'm running KCL on Windows 10 and ran into similar issues and thankfully wasn't stuck for long.

You can use either an absolute or relative path in the kcl.properties file, these work for me:

```
executableName = dotnet "E:\\Absolute\\Path\\YourConsumer.dll"

executableName = dotnet "..\\bin\\Debug\\netcoreapp2.2\\YourConsumer.dll"
```

I haven't checked this running on Linux, so it may need tweaked.
